ecs-cli/modules/commands/flags.go

```diff
@@ -48,6 +48,7 @@ const (
 	SubnetIdsFlag                   = "subnets"
 	VpcIdFlag                       = "vpc"
 	InstanceTypeFlag                = "instance-type"
+	InstanceRoleFlag                = "role"
```

Comment:
Yeah, `instance-role` would probably be best. Originally @yinshiua wanted the flag to just be `--role` but I agree it's better to be explicit. Will fix!